graylog2-server/src/main/java/org/graylog2/plugin/inputs/MessageInput.java

```diff
@@ -345,6 +348,7 @@ public void processRawMessage(RawMessage rawMessage) {
         rawMessage.setCodecName(codec.getName());
         rawMessage.setCodecConfig(codecConfig);
         rawMessage.addSourceNode(getId(), serverStatus.getNodeId());
+        rawMessage.setSequenceNr(sequenceNr.incrementAndGet());
```

Comment:
Using `.getAndIncrement()` would give us one more sequence number because we would start at `0`. :wink: